tools/test_configs/config_paths.json

```diff
@@ -4,9 +4,10 @@
     "HEAPBLOCKDEVICE_AND_ETHERNET": "HeapBlockDeviceAndEthernetInterface.json",
     "ODIN_WIFI" : "OdinInterface.json",
     "ODIN_ETHERNET" : "Odin_EthernetInterface.json",
-    "REALTEK_WIFI" : "RealtekInterface.json",	
+    "REALTEK_WIFI" : "RealtekInterface.json",   
```

Comment:
There's some whitespace characters at the end of line, otherwise loogs good to me